packages/frameworks/src/frameworks.ts

```diff
@@ -1772,7 +1772,7 @@ export const frameworks = [
 
       return (config && config.publishDir) || 'public';
     },
-    defaultVersion: '0.92.2',
+    defaultVersion: '0.64.0',
```

Comment:
This version isn't correct. The default version shipped in the build image is `0.58.2`

Reply:
Hmm - maybe this is the root issue - where is that at?